packages/flutter/lib/src/rendering/paragraph.dart

```diff
@@ -465,6 +465,9 @@ class RenderParagraph extends RenderBox
   // specified.
   void _layoutChildren(BoxConstraints constraints) {
     RenderBox child = firstChild;
+    if (childCount == 0) {
```

Comment:
Why not do this as the first thing in the method?

Reply:
Oops that's what I meant it to do.